build-system/amp4test.js

```diff
@@ -96,13 +96,16 @@ const bank = {};
  * Deposit a request. An ID has to be specified. Will override previous request
  * if the same ID already exists.
  */
-app.use('/request-bank/deposit/:id/', (req, res) => {
+app.use('/request-bank/:bid/deposit/:id/', (req, res) => {
+  if (!bank[req.params.bid]) {
+    bank[req.params.bid] = {};
+  }
   const key = req.params.id;
   log('SERVER-LOG [DEPOSIT]: ', key);
```

Comment:
Since id is a number now, I don't think it's useful to log the id anymore.

Reply:
still good to have a line printed when something gets deposited. it helped a lot when I was debugging.

Reply:
Same, I would argue that id can be a string.